### ISSUE TITLE
Create a screen reader accessible autocomplete

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wcag-em-report-tool#101]
-- Feedback by Bim Egan:
"Web technology" and the "Related specification or URI" fields should be auto complete.  Currently not picked up as such by screen readers. In addition they appear to use an onChange event, which isn't accessible for any keyboard only IE user.

**Comment by WilcoFiers** [comment]
As of 1.1 this issue is no longer relevant. The way that relied upon technologies are selected is changed and now doesn't use datalist anymore.
